trace-utils/src/trace_utils.rs

```diff
@@ -149,7 +109,7 @@ pub fn construct_agent_payload(tracer_payloads: Vec<pb::TracerPayload>) -> pb::A
 
 pub fn construct_trace_chunk(trace: Vec<pb::Span>) -> pb::TraceChunk {
     pb::TraceChunk {
-        priority: 1,
+        priority: i8::MIN as i32, // SamplerPriority::None
```

Comment:
I assume based on the comment that `SamplerPriority::None` is `i8::MIN` -- is there some reason we can't just use `SamplerPriority::None`?

Reply:
switched over to using SamplerPriority enum from the normalizer crate